src/app/modules/test-navigator/test-navigator.component.ts

```diff
@@ -66,6 +66,14 @@ export class TestNavigatorComponent implements OnInit, OnDestroy {
     indicatorFields: []
   };
 
+  getFilteredOutMarkers = (type: FilterType) => {
```

Comment:
```suggestion
  /* get markers that are not visible in the test navigator, since affected files are filtered by the current FilterState */
  getFilteredOutMarkers = (type: FilterType) => {
```